app/src/main/java/de/westnordost/streetcomplete/quests/fuel_service/AddFuelSelfService.kt

```diff
@@ -0,0 +1,41 @@
+package de.westnordost.streetcomplete.quests.fuel_service
+
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.data.osm.edits.update_tags.StringMapChangesBuilder
+import de.westnordost.streetcomplete.data.osm.osmquests.OsmFilterQuestType
+import de.westnordost.streetcomplete.data.quest.NoCountriesExcept
+import de.westnordost.streetcomplete.data.user.achievements.QuestTypeAchievement.CAR
+import de.westnordost.streetcomplete.ktx.toYesNo
+import de.westnordost.streetcomplete.quests.YesNoQuestAnswerFragment
+
+class AddFuelSelfService : OsmFilterQuestType<Boolean>() {
+
+    override val elementFilter = """
+        nodes, ways with
+          amenity = fuel
+          and !self_service
+          and !automated
+    """
+    override val commitMessage = "Add self service information to fuel station"
+    override val wikiLink = "Tag:amenity=fuel"
+    override val icon = R.drawable.ic_fuel_self_service
```

Comment:
The icon should be called `ic_quest_fuel_self_service` to be consistent with other quests.